pkgs/shells/zsh-prezto/default.nix

```diff
@@ -1,28 +1,14 @@
 { stdenv, fetchpatch, fetchgit, fetchFromGitHub }:
 
-let
-  # https://github.com/spwhitt/nix-zsh-completions/pull/2
-  nix-zsh-completions = fetchFromGitHub {
-    owner = "garbas";
-    repo = "nix-zsh-completions";
-    rev = "9b7d216ec095ccee541ebfa5f04249aa2964d054";
-    sha256 = "1pvmfcqdvdi3nc1jm72f54mwf06yrmlq31pqw6b5fczawcz02jrz";
-  };
-in stdenv.mkDerivation rec {
-  rev = "4f19700919c8ebbaf75755fc0d03716d13183f49";
+stdenv.mkDerivation rec {
+  rev = "221c6cd1283654ba77db052ee27aef613395321c";
   name = "zsh-prezto-2015-03-03_rev${builtins.substring 0 7 rev}";
```

Comment:
Can you also update the date and remove the revision?

Reply:
tnx, makes sense, i'll fix it